components/molecules/m-product-additional-info.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <SfTabs class="m-product-additional-info" :open-tab="2">
+    <SfTab title="Description">
+      <div>
+        <p>
+          The Karissa V-Neck Tee features a semi-fitted shape that's flattering
```

Comment:
I guess it shouldn't be static :)

Reply:
Yes :) I will finish this in issue 49 and 50. :) Here I focused on 48 and basic 49,50 with dummy data.